### DISCUSSION TITLE
Button click retried automatically

### DISCUSSION THREAD

**Discussion by readbeard** [mobile-dev-inc/Maestro#52] [Q&A] [answered]
Hey there! 

Is there a way to disable (even temporarily) the automatic multiple clicks on buttons to handle flakiness?

**Comment by dmitry-zaitsev** [comment, marked as answer]
This was addressed in one of the most recent versions of Maestro.

Alternatively, you can disable it like that:

```
- tapOn:
       text: Hello World
       retryTapIfNoChange: false
```
